.env.dev.tpl

```
@@ -95,6 +95,25 @@ STAGING_MOCK_IDP_AUTHORIZATION_ENDPOINT=https://localhost:9091/authorization?org
 STAGING_MOCK_IDP_TOKEN_ENDPOINT=https://localhost:9091/token
 STAGING_MOCK_IDP_USERINFO_ENDPOINT=https://localhost:9091/userinfo
 
+# =============================================================================
+# Account Management Settings (ローカル開発用ダミー値)
+# =============================================================================
+# AccountsOrganizationSeeder が accounts / stg-accounts Org と管理コンソール Client を
+# 投入するために参照する。本番では Terraform app_settings (Phase E) から注入される。
+# Organization の code / tenant_name はサブドメイン解決と一致させるため Seeder 側で固定。
+ACCOUNTS_CLIENT_ID=ecauth-admin-console
+ACCOUNTS_CLIENT_SECRET=accounts_client_secret
+ACCOUNTS_ALLOWED_RP_IDS=localhost
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ACCOUNTS_ALLOWED_RP_IDS key should go before the ACCOUNTS_CLIENT_ID key

(UnorderedKey)